backend/tests/unit/core/test_annotation_unit.py

```diff
@@ -10,13 +11,60 @@ def test_annotation_unit_gets_missing_dependencies(annotation_unit_lmna):
     assert actual == ['HGNC_ID']
 
 
+def test_annotation_unit_ready_for_annotation(annotation_unit_lmna):
+    """Verifies if the annotation unit is ready for annotation"""
+
+    missing = annotation_unit_lmna.get_missing_dependencies()
+    missing_dependency = missing[0]
+    mock_genomic_unit_collection = Mock()
+    mock_genomic_unit_collection.find_genomic_unit_annotation_value = Mock()
+    dependency_annotation = mock_genomic_unit_collection.find_genomic_unit_annotation_value(
+        annotation_unit_lmna.genomic_unit, missing_dependency
+    )
+
+    actual = annotation_unit_lmna.ready_for_annotation(dependency_annotation, missing_dependency)
+    assert actual is True
+
+    dependency_annotation = ""
```

Comment:
It will be the same for this test case as I commented in the unit test after this.

Am requesting this part of the unit test but in a separate unit test function.  That way, each of these cases can get individually documented in the unit test results.